xbmc/ServiceManager.h

```diff
@@ -113,6 +113,7 @@ class CServiceManager
   XBPython& GetXBPython();
 #endif
   PVR::CPVRManager& GetPVRManager();
+  bool HasActiveAE() const;
```

Comment:
Should we add a delegate to our service broker so we are able to drop the use of `g_application.m_ServiceManager`?

Reply:
WTF was unclear about my pr description?

Reply:
and that is not straight forward as shit use the service manager directly elsewhere so then i have to change those spots from . to ->.

Reply:
Ok sorry for the noise

Reply:
nw. i spewed some frustrations out on you which was not fair, sorry for that.

Reply:
Np .. all good.